gym-unity/gym_unity/envs/__init__.py

```diff
@@ -18,9 +18,8 @@ class UnityGymException(error.Error):
     pass
 
 
-logging.basicConfig(level=logging.INFO)
-logger = logging.getLogger("gym_unity")
-
+logger = get_logger(__name__)
```

Comment:
Recommended logging practice is to use `__name__` for the logger name: https://docs.python.org/3.7/library/logging.html#logger-objects